install-env.sh

```diff
@@ -11,10 +11,9 @@ cp certs/mainflux-server.key /etc/ssl/private/
 openssl dhparam -out /etc/ssl/certs/dhparam.pem 2048
 
 # Copy nginx config snippets:
+mkdir -p /etc/nginx/snippets
 cp snippets/* /etc/nginx/snippets/
 
-# Copy `sites-avalable/mainflux-proxy`:
-cp sites-available/mainflux-proxy /etc/nginx/sites-available/mainflux-proxy
-
-# Enable it:
-ln -s /etc/nginx/sites-available/mainflux-proxy /etc/nginx/sites-enabled/mainflux-proxy
+# Enable proxy as default:
+mv /etc/nginx/conf.d/default.conf /etc/nginx/conf.d/default.conf.bak
+cp conf.d/mainflux-proxy.conf /etc/nginx/conf.d/mainflux-proxy.conf
```

Comment:
I looked a little further and it seems that the sites-enabled pattern was actually added by debian distros. I think latest ubuntu packages did away with it but not sure about others. Its not a requirement of nginx though. Nginx just works by telling it where to load configuration files (the `include` statements) so in this example we're just removing the superfluous extra directories `sites-*` and using the `conf.d` directory that already existed.

Heres the layout of the latest alpine image
```
❯ docker run --rm nginx:alpine ls -la /etc/nginx
total 72
drwxr-xr-x    4 root     root          4096 May 10 21:50 .
drwxr-xr-x   20 root     root          4096 May 21 16:39 ..
drwxr-xr-x    2 root     root          4096 May 10 21:50 conf.d
-rw-r--r--    1 root     root          1077 May 10 21:50 fastcgi.conf
-rw-r--r--    1 root     root          1077 May 10 21:50 fastcgi.conf.default
-rw-r--r--    1 root     root          1007 May 10 21:50 fastcgi_params
-rw-r--r--    1 root     root          1007 May 10 21:50 fastcgi_params.default
-rw-r--r--    1 root     root          2837 May 10 21:50 koi-utf
-rw-r--r--    1 root     root          2223 May 10 21:50 koi-win
-rw-r--r--    1 root     root          3957 May 10 21:50 mime.types
-rw-r--r--    1 root     root          3957 May 10 21:50 mime.types.default
lrwxrwxrwx    1 root     root            27 May 10 21:50 modules -> ../../usr/lib/nginx/modules
-rw-rw-r--    1 root     root           643 May 10 21:49 nginx.conf
-rw-r--r--    1 root     root          2656 May 10 21:50 nginx.conf.default
-rw-r--r--    1 root     root           636 May 10 21:50 scgi_params
-rw-r--r--    1 root     root           636 May 10 21:50 scgi_params.default
-rw-r--r--    1 root     root           664 May 10 21:50 uwsgi_params
-rw-r--r--    1 root     root           664 May 10 21:50 uwsgi_params.default
-rw-r--r--    1 root     root          3610 May 10 21:50 win-utf
```

```
❯ docker run --rm nginx:alpine ls -la /etc/nginx/conf.d
total 12
drwxr-xr-x    2 root     root          4096 May 10 21:50 .
drwxr-xr-x    4 root     root          4096 May 10 21:50 ..
-rw-rw-r--    1 root     root          1097 May 10 21:49 default.conf
```